modules/subchannel/test/tests/problems/SFR/EBR-II/XX09_SS_SHRT17.i

```diff
@@ -210,8 +211,75 @@ unheated_length_exit = '${fparse 26.9*scale_factor}'
   []
 []
 
+[Postprocessors]
+  [TTC-27]
+    type = SubChannelPointValue
+    variable = T
+    index = 91
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-28]
+    type = SubChannelPointValue
+    variable = T
+    index = 50
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-29]
+    type = SubChannelPointValue
+    variable = T
+    index = 21
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-30]
+    type = SubChannelPointValue
+    variable = T
+    index = 4
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-31]
+    type = SubChannelPointValue
+    variable = T
+    index = 2
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-32]
+    type = SubChannelPointValue
+    variable = T
+    index = 16
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-33]
+    type = SubChannelPointValue
+    variable = T
+    index = 42
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-34]
+    type = SubChannelPointValue
+    variable = T
+    index = 80
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+  [TTC-35]
+    type = SubChannelPointValue
+    variable = T
+    index = 107
+    execute_on = 'TIMESTEP_END'
+    height = 0.322
+  []
+[]
```

Comment:
let's test more than temperature here. Maybe pressure and mass flow?

Imo that would be sufficient:
- All the other quantities are tied to temperature (rho, h, mu, q_prime) or geometry (S, w_)
- P & dP -> covered by pressure
- Wij -> unreliable

Reply:
just to make sure we have everything, did you publish on this case? 
and if so what did you compare ?